src/osha/oira/statistics/utils.py

```diff
@@ -84,6 +77,16 @@ def update_database(self, country=None):
         self.log_counts()
 
     def update_tool(self, country=None):
+        log.info("Table: tool")
+        latest_published_date = (
+            self.session_statistics.query(
+                sqlalchemy.func.max(SurveyStatistics.published_date)
+            ).first()[0]
+            or datetime.min
+        )
+        if latest_published_date > datetime.min:
+            log.info(f"Skipping tools up to and including {latest_published_date}")
```

Comment:
AFAIK calling the logger with f-strings is not encouraged.
It is fine to merge this but, please, do not get used to it.

This is better:
```
log.info("Skipping tools up to and including %r", latest_published_date)
```
The reason is that f-strings are computed before the call, while in the "old" way I propose the message creation is executed only if the message will be logged (e.g. if the loglevel is high enough).

Reply:
Thanks for the reminder. This apparently slipped past both my attention and my linter.